pkg/controllers/secretsync/secret_sync_controller.go

```diff
@@ -176,59 +158,59 @@ func (r *UserDataSecretController) SetupWithManager(mgr ctrl.Manager) error {
 		).
 		Watches(
 			&corev1.Secret{},
-			handler.EnqueueRequestsFromMapFunc(toUserDataSecret),
-			builder.WithPredicates(userDataSecretPredicate(SecretSourceNamespace)),
+			handler.EnqueueRequestsFromMapFunc(toUserDataSecret(r.SourceNamespace)),
+			builder.WithPredicates(userDataSecretPredicate(r.SourceNamespace)),
 		).
 		Complete(r); err != nil {
-		return fmt.Errorf("failed to create controller: %w", err)
+		return fmt.Errorf("failed to create secret sync controller: %w", err)
 	}
 
 	return nil
 }
 
-func (r *UserDataSecretController) setAvailableCondition(ctx context.Context, log logr.Logger) error {
+// setControllerConditionsToNormal sets the SecretSyncController conditions to the normal state.
+func (r *SecretSyncController) setControllerConditionsToNormal(ctx context.Context, log logr.Logger) error {
 	co, err := r.GetOrCreateClusterOperator(ctx)
 	if err != nil {
-		return fmt.Errorf("unable to get cluster operator: %w", err)
+		return fmt.Errorf("failed to get cluster operator: %w", err)
 	}
 
 	conds := []configv1.ClusterOperatorStatusCondition{
-		operatorstatus.NewClusterOperatorStatusCondition(secretSyncControllerAvailableCondition, configv1.ConditionTrue, operatorstatus.ReasonAsExpected,
-			"User Data Secret Controller works as expected"),
-		operatorstatus.NewClusterOperatorStatusCondition(secretSyncControllerDegradedCondition, configv1.ConditionFalse, operatorstatus.ReasonAsExpected,
-			"User Data Secret Controller works as expected"),
+		operatorstatus.NewClusterOperatorStatusCondition(operatorstatus.SecretSyncControllerAvailableCondition, configv1.ConditionTrue, operatorstatus.ReasonAsExpected,
+			"Secret Sync Controller Controller works as expected"),
+		operatorstatus.NewClusterOperatorStatusCondition(operatorstatus.SecretSyncControllerDegradedCondition, configv1.ConditionFalse, operatorstatus.ReasonAsExpected,
+			"Secret Sync Controller Controller works as expected"),
 	}
 
-	co.Status.Versions = []configv1.OperandVersion{{Name: controllers.OperatorVersionKey, Version: r.ReleaseVersion}}
-
-	log.Info("user Data Secret Controller is available")
+	log.V(2).Info("Secret Sync Controller Controller is Available")
 
 	if err := r.SyncStatus(ctx, co, conds); err != nil {
-		return fmt.Errorf("failed to sync status: %w", err)
+		return fmt.Errorf("failed to sync cluster operator status: %w", err)
 	}
 
 	return nil
 }
 
-func (r *UserDataSecretController) setDegradedCondition(ctx context.Context, log logr.Logger) error {
+// setControllerConditionDegraded sets the UserDataSecretController conditions to a degraded state.
+//
+//nolint:unused
```

Comment:
If it's not used, drop it

Looks like perhaps previously it was used? So why do we no longer have a degraded state for this controller?